R/createQuadrantGate.R

```diff
@@ -39,12 +39,13 @@
 #' createQuadrantGate(experimentId, "FSC-A", "FSC-W", "my gate", 160000, 200000)
 #' }
 createQuadrantGate = function(experimentId, xChannel, yChannel, name,
-                               x, y, angles = c(0.000000, 3/2*pi, pi, pi/2),
+                               x, y, angles = c(0.000000, 3/2*pi, pi, pi/2), labels = list(),
```

Comment:
Maybe default `labels` this to `NULL`? (Semantically that's like "unspecified" vs. a nonsense value.)